internal/errs/exitcode.go

```diff
@@ -7,10 +7,11 @@ type ExitCodeable interface {
 type ExitCode struct {
 	code       int
 	wrappedErr error
+	silent     bool
 }
 
-func WrapExitCode(err error, code int) error {
-	return &ExitCode{code, err}
+func WrapExitCode(err error, code int, silent bool) error {
```

Comment:
This is overloading the meaning of WrapExitCode. Its responsibility is wrapping exit codes, full stop. I your use-cases for this should instead also invoke `Silence()`.